scripts/core/EXT.rst

```diff
@@ -0,0 +1,91 @@
+<%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+==============
+ Extensions
+==============
+
+Objective
+=========
+
+Extensions allow for additional functionality to be added to the Unified Runtime specification
+without affecting the core specification. Extensions may in future be promoted to the core specification
+in a later version of Unified Runtime, when agreed upon by the Working Group. All extensions are 
+optional and are not required to be implemented by any particular adapter, but are expected to 
+be widely available and possibly be required in a future version of the specification. Adapters 
+must report which extensions are supported through the ${x}PlatformGetExtensionProperties query.
+Each extensions may also impose additional restrictions on when it can be used - i.e. a platform
+or device query.
+
+There are two types of extensions defined by this specification:
+
+1. **Standard** -  extension will be included into the current and all future version of the specification.
+2. **Experimental** - extensions require additional experimentation and development, before becoming a standard extension. 
+Applications should not rely on experimental extensions in production.
+
+Requirements
+============
+
+- Extensions must use globally unique names for macros, enums, structures and functions
+- Extensions must have globally unique extension names reported from ${x}PlatformGetExtensionProperties
+- All extensions must be defined in this specification
+- Extensions must not break backwards compatibility of the core APIs
+- Standard extension versions must be backwards compatible with prior versions
```

Comment:
I copied this from L0 extension scheme, but perhaps we could use semantic versioning with extensions with major/minor versions?